client/my-sites/stats/stats-module/index.jsx

```diff
@@ -120,6 +212,36 @@ class StatsModule extends Component {
 		return summary && includes( summarizedTypes, statType );
 	}
 
+	remapData() {
+		const { valueField, query } = this.props;
+		const isRealTime = query?.interval !== undefined;
```

Comment:
[As suggested](https://github.com/Automattic/wp-calypso/pull/99550/files#diff-d73e6c6c40efc0c3e95fbac3a82b9ce7065e95acd639f4aace88ce7f649a2211R44), could we pass `isRealTime` as a new prop to avoid attaching an interval to the query object?

Reply:
Yes, I'll revisit this. Due to the way the loading and empty states are handled, I might have to move the request management up to the page level anyway. When testing on a site with little traffic, the empty state is displayed initially and because the `StatsModule` is never presented, the query is never triggered again...